cfgov/v1/util/ref.py

```diff
@@ -266,3 +266,12 @@ def get_category_children(category_names):
         dict(categories_dict[category]).keys()
         for category in category_names
     )))
+
+def get_fl_display_settings(filterable_page_type):
+    settings = {
+        'page_type': filterable_page_type,
+        'show_tags': True,
+        }
+    if "foia-freq-req-record" in filterable_page_type:
```

Comment:
Should this be `if "foia-freq-req-record" == filterable_page_type`?

Reply:
I used `in` because it is used elsewhere in similar cases. There are also some extraneous characters in the output, so I think `in` was the reliable choice here. I'm all ears to best practice on this!

Reply:
I'm not sure either, but I thought `in` probably refers to a collection, so array or object. Whereas `==` would be string comparison. You may want to do string comparison with whitespace trimming, e.g.`filterable_page_type.strip()`.